_pages/documentation/learning_gem5/part2/helloobject.md

```diff
@@ -57,7 +57,7 @@ and the C++ header that will define the C++ class for the SimObject.
 
 We can create a file, HelloObject.py, in `src/learning_gem5`
 
-``` {.sourceCode .python}
+```
```

Comment:
We can specify the language after the "\`\`\`" E.g., "\`\`\`python"

Reply:
Yeah. I've looked it up. The github pages syntax is
"```"python
...
"```"

I'll update this.

Reply:
Thanks!